packages/neutron-sdk/src/bindings/msg.rs

```diff
@@ -195,12 +198,28 @@ impl NeutronMsg {
         query_id: u64,
         new_keys: Option<Vec<KVKey>>,
         new_update_period: Option<u64>,
-    ) -> Self {
-        NeutronMsg::UpdateInterchainQuery {
+        new_transactions_filter: Option<Vec<TransactionFilterItem>>,
+    ) -> NeutronResult<Self> {
+        Ok(NeutronMsg::UpdateInterchainQuery {
             query_id,
             new_keys,
             new_update_period,
-        }
+            new_transactions_filter: match new_transactions_filter {
+                Some(filters) => {
+                    if filters.len() > MAX_TX_FILTERS {
```

Comment:
do we really need to validate filters len here? neutron icq module does it, why should we do it here? if the max possible value changes in the neutron, we'll have to take care of it here as well

Reply:
MAX_TX_FILTERS has been introduced not in this PR so I believe this case is out of scope of this PR.
Anyway I don't see any issue about handling this value in neutron and here, bc we already have to handle really many things like message formats (bindings) and etc